src/classifiers/transformer_based/predict.py

```diff
@@ -30,36 +32,26 @@
     print(f"Something went wrong with audio, exception: {e}")
     exit(1)
 
-print("  Transcribed text: {}".format(transcription))
-
 device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
 
-model = custom_model = CustomBertClassifier(
-    model_name=MODEL_NAME,
-    dropout_prob=DROPOUT_PROB,
-    n_classes=len(Y_CLASSES)
-)
-model.load_state_dict(torch.load(args.state))
+model_interface = BertClassifierInterface(args.model_type, DROPOUT_PROB, len(Y_CLASSES))
```

Comment:
Interface is only abstraction and it should not be created object from AbstractClass. You should use Factory class to create concrete Bert object.